app/views/card_grants/_details.html.erb

```diff
@@ -1,76 +1,93 @@
-<%# locals: (card_grant:) %>
```

Comment:
Generally it's best to use locals in partials and avoid instance variables: https://andycroll.com/ruby/dont-use-instance-variables-in-partials/. Yes I know our codebase commits this sin many times but I don't want to make it worse.